static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -298,20 +298,27 @@ function Content({
   const platformIcon = stackTracePlatformIcon(platform, data.frames ?? []);
 
   return (
-    <Wrapper className={wrapperClassName} data-test-id="stack-trace-content">
+    <Wrapper>
       {!hideIcon && <StacktracePlatformIcon platform={platformIcon} />}
-      <GuideAnchor target="stack_trace">
-        <StyledList data-test-id="frames">
-          {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
-        </StyledList>
-      </GuideAnchor>
+      <ContentPanel className={wrapperClassName} data-test-id="stack-trace-content">
+        <GuideAnchor target="stack_trace">
+          <StyledList data-test-id="frames">
+            {!newestFirst ? convertedFrames : [...convertedFrames].reverse()}
+          </StyledList>
+        </GuideAnchor>
+      </ContentPanel>
     </Wrapper>
   );
 }
 
-const Wrapper = styled(Panel)`
+const Wrapper = styled('div')`
+  position: relative;
+`;
+
+const ContentPanel = styled(Panel)`
   position: relative;
   border-top-left-radius: 0;
+  overflow: hidden;
```

Comment:
This fixes border radius issues on the last frame when the content is expanded. There was an existing issue where the frame was rounded, but it wasn't very visible since the frame background wasn't very different from the page background, but it became more visible with the hover state.